internal/cloud/atlas/groups.go

```diff
@@ -17,27 +19,67 @@ type Group struct {
 	Name string `json:"name"`
 }
 
-type groupResponse struct {
+// Link contains the url to the next for a set of groups and its current position in the chain
+type Link struct {
+	Href string `json:"href"`
+	Rel  string `json:"rel"`
+}
+
+// Groups represent the list of groups retrieved along with the links to retrieve more
```

Comment:
[meganit/optional] I usually go dumb/naive with these types of docstrings
```suggestion
// Groups represents the groups response from Atlas
```